docs/network/interfaces_and_networks.md

```diff
@@ -1114,7 +1007,79 @@ networks:
   name: br10
 ```
 
-#### Limitations
+## Limitations and known issues
+### Invalid CNIs for secondary networks
+The following list of CNIs is known **not** to work for bridge interfaces -
+which are most common for secondary interfaces.
+
+- [macvlan](https://www.cni.dev/plugins/current/main/macvlan/)
+
+- [ipvlan](https://www.cni.dev/plugins/current/main/ipvlan/)
+
+The reason is similar: the bridge interface type moves the pod interface MAC
+address to the VM, leaving the pod interface with a different address. The
+aforementioned CNIs require the pod interface to have the original MAC address.
+
+These issues are tracked individually:
+
+- [macvlan](https://github.com/kubevirt/kubevirt/issues/5483)
+
+- [ipvlan](https://github.com/kubevirt/kubevirt/issues/7001)
```

Comment:
Are these issues still relevant?

Reply:
Their status is `Closed`.

Reply:
indeed, but the problem has not been solved and we still recommend to avoid those CNIs